magpie/alembic/versions/24da162a54f1_ensure_service_root_is_none.py

```diff
@@ -38,6 +35,6 @@ def downgrade():
     # pylint: disable=no-member
     op.execute(resources.
                update().
-               where(resources.c.root_service_id == None).
+               where(resources.c.root_service_id is None).
```

Comment:
`==` is the actual syntax for sqlalchemy to produce `IS NULL`: https://docs.sqlalchemy.org/en/13/core/sqlelement.html#sqlalchemy.sql.expression.ColumnElement.__eq__

Reply:
done via `is_(None)`